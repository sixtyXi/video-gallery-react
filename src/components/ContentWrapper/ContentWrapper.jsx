// @flow
import * as React from 'react';

import styles from './ContentWrapper.scss';

type Props = {
  children: React.Node | null
};

export const ContentWrapper = ({ children }: Props) => {
  return <section className={styles.contentWrapper}>{children}</section>;
};

ContentWrapper.defaultProps = {
  children: null
};
