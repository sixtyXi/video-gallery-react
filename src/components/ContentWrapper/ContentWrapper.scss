.contentWrapper {
  max-width: 800px;
  flex-grow: 1;
  display: flex;
  align-items: flex-start;
  margin: 0 auto;
  padding: 0 40px;
}
