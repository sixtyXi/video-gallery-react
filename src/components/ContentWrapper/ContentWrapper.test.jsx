import React from 'react';
import { shallow } from 'enzyme';

import { ContentWrapper } from './ContentWrapper';

describe('ContentWrapper', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<ContentWrapper />);

    expect(wrapper).toMatchSnapshot();
  });
});
