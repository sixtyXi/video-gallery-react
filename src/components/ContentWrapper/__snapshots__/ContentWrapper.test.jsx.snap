// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentWrapper renders correctly 1`] = `
<section
  className="contentWrapper"
/>
`;
